core/templates/dev/head/components/top_navigation_bar/TopNavigationBarDirective.js

```diff
@@ -67,24 +82,61 @@ oppia.directive('topNavigationBar', [
             $window.localStorage.removeItem('last_uploaded_audio_lang');
           };
 
-          $scope.profileDropdownIsActive = false;
-          $scope.onMouseoverProfilePictureOrDropdown = function(evt) {
-            angular.element(evt.currentTarget).parent().addClass('open');
-            $scope.profileDropdownIsActive = true;
-          };
-          $scope.onMouseoutProfilePictureOrDropdown = function(evt) {
-            angular.element(evt.currentTarget).parent().removeClass('open');
-            $scope.profileDropdownIsActive = false;
-          };
-          $scope.onMouseoutDropdownMenuAbout = function(evt) {
-            angular.element(evt.currentTarget)[0].blur();
+          /**
+           * opens the submenu.
+           * @param {object} evt
+           * @param {String} menu - name of menu, on which
+           * open/close action to be performed (aboutMenu,profileMenu).
+           */
+          $scope.openSubmenu = function(evt, menu) {
+            //focus the current target before opening it's submenu.
```

Comment:
// Focus on the current target before opening its submenu.

(no apostrophe, capitalize, one space after //)